src/main/java/gov/nih/nci/evs/api/service/ElasticOperationsServiceImpl.java

```diff
@@ -139,4 +143,18 @@ public String delete(String indexName, String id) {
   public String deleteIndexMetadata(String id) {
     return operations.delete(id, IndexCoordinates.of(METADATA_INDEX));
   }
+
+  /** */
+  @Override
+  public Boolean deleteQuery(String query, String indexName) {
```

Comment:
That's reasonable, though the other deletes use a different version of operations.delete that returns a string by default (the ID of the document successfully deleted), while this one uses a version that returns a different object, a ByQueryResponse. I'll change it so that it returns the number of deleted documents, which seems like the most useful information from that object as far as I can tell.